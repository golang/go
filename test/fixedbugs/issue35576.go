// run

// Copyright 2019 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Check print/println(f()) is allowed where f() is multi-value.

package main

func f() (int16, float64, string) { return -42, 42.0, "x" }

func main() {
	print(f())
	println(f())
}
