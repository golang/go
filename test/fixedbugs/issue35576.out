-42+4.200000e+001x-42 +4.200000e+001 x
